core/renderers/zelos/measurables/inputs.js

```diff
@@ -10,33 +10,34 @@
  * @author samelh@google.com (Sam El-Husseini)
  */
 
-goog.provide('Blockly.zelos.StatementInput');
+goog.module('Blockly.zelos.StatementInput');
+goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.blockRendering.StatementInput');
-goog.require('Blockly.utils.object');
-
-goog.requireType('Blockly.blockRendering.ConstantProvider');
-goog.requireType('Blockly.Input');
+const BaseStatementInput = goog.require('Blockly.blockRendering.StatementInput');
+/* eslint-disable-next-line no-unused-vars */
+const ConstantProvider = goog.requireType('Blockly.blockRendering.ConstantProvider');
+/* eslint-disable-next-line no-unused-vars */
+const Input = goog.requireType('Blockly.Input');
+const object = goog.require('Blockly.utils.object');
 
 
 /**
  * An object containing information about the space a statement input takes up
  * during rendering
- * @param {!Blockly.blockRendering.ConstantProvider} constants The rendering
+ * @param {!ConstantProvider} constants The rendering
  *   constants provider.
- * @param {!Blockly.Input} input The statement input to measure and store
+ * @param {!Input} input The statement input to measure and store
  *     information for.
  * @package
  * @constructor
- * @extends {Blockly.blockRendering.StatementInput}
+ * @extends {BaseStatementInput}
  */
-Blockly.zelos.StatementInput = function(constants, input) {
-  Blockly.zelos.StatementInput.superClass_.constructor.call(this,
-      constants, input);
+const StatementInput = function(constants, input) {
+  StatementInput.superClass_.constructor.call(this, constants, input);
 
   if (this.connectedBlock) {
     // Find the bottom-most connected block in the stack.
-    var block = this.connectedBlock;
+    let block = this.connectedBlock;
     while (block.getNextBlock()) {
       block = block.getNextBlock();
```

Comment:
Yuck, this code sucks.  It's not a result of this PR, but that's a horrible repeated call.

Consider using a variable to cache the call:
```
let nextBlock;
while ((nextBlock = block.getNextBlock())) {
      block = nextBlock;
```
Either add it to this PR, or if you feel religious about "one change per PR", follow up with a separate PR.  Thanks!